src/test/skript/tests/misc/literal multiple infos warning.sk

```diff
@@ -0,0 +1,57 @@
+options:
+	error: "'unknown' has multiple types. Consider specifying which type to use: 'unknown (inventory action)'"
+
+using for each loops
+
+parse:
+	results: {LiteralMultipleWarn::FunctionObject}
+	code:
+		function a():: object:
+			return unknown
+
+test "literal multiple infos warning":
+	parse:
+		set {_test} to unknown
+	assert last parse logs contains {@error} with "Setting variable should throw warning"
+
+	parse:
+		if {_list::*} contains unknown:
+			broadcast "Filler"
+	assert last parse logs contains {@error} with "Contains should throw warning"
+
+	parse:
+		if the 1st element of {_list::*} is unknown:
+			broadcast "filler"
+	assert last parse logs contains {@error} with "Element of should throw warning"
+
+	parse:
+		filter {_list::*} to match:
+			input is unknown
```

Comment:
i'd like to also see this tested with a specified input, like `damage cause input is unknown` and verify there is no warning